www/ondv03/style.css

```diff
@@ -0,0 +1,91 @@
+body{
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.text#first{
+    color: red
+}
+.text#second{
+    color: rgb(0, 128, 0)
+}
+.text#third{
+    color: #009990;
+    font-weight: bold;    
+}
+.text#fourth{
+    font-size: 18px
+}
+.text#fifth{
+    font-size: 24px;
+    font-style: italic
+}
+.text#sixth{
+    color: white;
+    background-color: rgb(128, 87, 0)
+}
+p#first{
+    text-align: center;
+    line-height: 32px
+}
+p#second{
+    text-justify: auto;
+    text-transform: uppercase;
+}
+.box#first{
+    width: 400px;
+    height: 300px;
+    border: 1px solid purple;
+    padding: 30px;
+}
+.box#second{
+    width: 500px;
+    height: 200px;
+    background-color: rgb(128, 0, 17);
+    margin: 100px;    
+}
+.text#seventh{
+    visibility: hidden;
+}
+.box#third{
+    width: auto;
+    height: 150px;
+    background-color: rgb(0, 81, 128);
+    padding: 50px;
+    border: 10px rgb(191, 224, 4);
+}
+.text#eighth{
+    visibility: hidden;
+}
+.box#fourth{
+    width: 200px;
+    height: 500px;
+    background-color: rgb(58, 133, 58);
+    opacity: 0.5;
+}
+.box#fifth{
+    width: 400px;
+    height: 300px;
+    position: relative;
+    background-color: lightblue;
+}
+.box#sixth{
+    width: 50px;
+    height: 50px;
+    position: absolute;
+    margin-top: 50px;
+    margin-left: 340px;
+    background-color: pink;
+}
+.button#first{
+    width: 300px;
+    height: 100px;
+}
+.button#first:active{
+    color: blue;
+}
+.button#second{
+    width: 200px;
+    height: 50px;
+}
+.button#second:hover{
+    cursor: -webkit-grab;
```

Comment:
vlastnost **cursor: pointer**, a nemusí to být v :hover, protože sama již implikuje to, že element musí být hovered, aby to mělo efekt